libvast/src/format/pcap.cpp

```diff
@@ -503,11 +503,11 @@ caf::error writer::write(const table_slice& slice) {
     return make_error(ec::format_error, "invalid pcap packet type");
   // TODO: consider iterating in natural order for the slice.
   for (size_t row = 0; row < slice.rows(); ++row) {
-    auto payload_field = slice.at(row, 6);
+    auto payload_field = slice.at(row, 6, *pcap_packet_type.at("payload"));
     auto& payload = caf::get<view<std::string>>(payload_field);
     // Make PCAP header.
     ::pcap_pkthdr header;
-    auto ns_field = slice.at(row, 0);
+    auto ns_field = slice.at(row, 0, *pcap_packet_type.at("time"));
```

Comment:
A bit unrelated: I never liked the manual integer offsets to get to a field. Maybe we can factor this logic to happen dynamically when the writer gets instantiated?

Reply:
I think that belongs to another PR.